src/basic/parse-util.c

```diff
@@ -566,6 +566,36 @@ int safe_atod(const char *s, double *ret_d) {
         return 0;
 }
 
+int safe_hexstring_aton(const char *s, uint8_t *b, unsigned int buf_len, unsigned int *len) {
+        unsigned int c = 0;
```

Comment:
I think c=0 is not necessary as it is always initialized later.

Reply:
yes right